extensions/xdebug/lib/XDebugUtil.php

```diff
@@ -13,10 +13,41 @@
 namespace PhpBench\Extensions\XDebug;
 
 use PhpBench\Executor\ExecutionContext;
+use RuntimeException;
 
 class XDebugUtil
 {
-    public static function filenameFromContext(ExecutionContext $context, $extension = ''): string
+    public static function fromEnvironment(): XDebugUtil
+    {
+        $xdebugVersion = phpversion('xdebug');
+
+        if ($xdebugVersion === false) {
+            throw new RuntimeException(
+                'Xdebug is not installed'
```

Comment:
![image](https://user-images.githubusercontent.com/530801/199187580-ba9e122c-2610-4924-945a-0f90b13e6708.png)

this is eagery loaded, so we do indeed need to accept `false|string` I think.